Library/Formula/ffmpegthumbnailer.rb

```diff
@@ -12,16 +11,52 @@ class Ffmpegthumbnailer < Formula
     sha1 "052f2227429e215db559dcbddb2cdca838111d59" => :lion
   end
 
-  revision 2
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "ffmpeg"
 
-  depends_on 'pkg-config' => :build
-  depends_on 'jpeg'
-  depends_on 'libpng'
-  depends_on 'ffmpeg'
+  patch :DATA
 
   def install
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make install"
+    # The CMake script does not allow fine-grained control of man directories or
+    # other installation paths beneath the prefix
+    inreplace "CMakeLists.txt", "${CMAKE_INSTALL_PREFIX}/man/man1", man1
+
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args, "-DENABLE_GIO=ON",
```

Comment:
Older versions of Ruby can't have anything in a list like this after an `*exploded` array. Just swapping `*std_cmake_args` with `"-DENABLE_GIO=ON"` should fix the ML build.